src/library/scanner/libraryscanner.cpp

```diff
@@ -99,11 +99,11 @@ LibraryScanner::LibraryScanner(QWidget* pParentWidget, TrackCollection* collecti
 }
 
 LibraryScanner::~LibraryScanner() {
-    // A scan is running.
-    if (m_scannerGlobal) {
-        // Cancel any running library scan.
-        cancel();
-    }
+    // Wait until there is no pending scan request
```

Comment:
Well, I know. 
My question is: what is the typical time this particular mutex is on (time of a pending scan request if I understand correctly the code).
We wait here for the mutex to be release elsewhere in the code (when there is no more pending request), so how long could this be ? if it's a matter of milliseconds, that's ok, but if there can be conditions where the request can be pending for seconds or minutes, that would delay Mixxx closing in a way that is disturbing to users.